### ISSUE TITLE
iPhone zooms in on input

### ISSUE THREAD

**Issue by AustinGil** [Desertsnowman/Caldera-Forms#544]
This isn't so much a bug with the plugin as much as a UI bug with iPhones. When clicking on an input with the font size smaller than 16px (the default font size for Caldera) the phone will zoom the screen in. 

The font size is applied to ".caldera-grid .form-control". Changing it to font size 16px or adding a break point for that would solve this issue.

**Comment by Shelob9** [comment]
I'm not sure this is a bug. Shouldn't it zoom in? I'm not very familiar with iOS, but isn't that a good thing on mobile? @Desertsnowman what do you think?

**Comment by AustinGil** [comment]
Yeah, like I said, it's not really a bug. But the problem is that after it zooms in, the screen does not automatically zoom out. This creates horizontal scrolling on the site which wouldnt otherwise be the case.
